### DISCUSSION TITLE
WebGIS5-CMS WFS-Dienst einbinden

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#9] [Q&A]
Das Einbindeneines WFS Dienstes ins CMS wird bei "Übernehmen" mit folgender Fehlermeldung quittiert:
NullReferenceException: Object reference not set to an instance of an object.

Die Kollegen Wasserwirtschaft haben im WebGIS4 alle Abfragen auf WFS aufgebaut und benötigen das auch in WebGIS5

**Comment by gis-eni** [comment]
Die WFS Schnittstelle wurde schon länger nicht mehr gepflegt, weil diese Dienste im WebGIS eher ein "Fremdkörper" sind. 
Dazu bräuchte ich mehr Details:
* Der Dienst als solcher (kannst du mir den Link per Mail schicken)
* Was willst du damit genau machen bzw. was erwartest du dir, das WebGIS für diesen Dienst im Viewer anbieten soll?

**Comment by gis-eni** [comment]
Dienst wurde geprüft und Bugs/fehlende Features im WebGIS behoben.
